translations/monero-core_de.ts

```diff
@@ -11,12 +11,13 @@
     <message>
         <location filename="../pages/AddressBook.qml" line="56"/>
         <source>Qr Code</source>
-        <translation type="unfinished"></translation>
+        <translation>QR-Code</translation>
     </message>
     <message>
         <location filename="../pages/AddressBook.qml" line="74"/>
         <source>4...</source>
-        <translation type="unfinished"></translation>
+        <translatorcomment>Placeholder value</translatorcomment>
+        <translation></translation>
```

Comment:
Don't leave empty fields. If you are not sure on how to translate a string, leave the tag `unfinished` so that the original string will get picked. Empy fields will result in an empty line